fe/src/main/java/org/apache/doris/alter/SchemaChangeJobV2.java

```diff
@@ -1081,14 +969,8 @@ private void readJobFinishedData(DataInput in) throws IOException {
 
     @Override
     public void write(DataOutput out) throws IOException {
-        super.write(out);
-
-        out.writeBoolean(isMetaPruned);
-        if (isMetaPruned) {
-            writeJobFinishedData(out);
-        } else {
-            writeJobNotFinishData(out);
-        }
+        String json = GsonUtils.GSON.toJson(this, AlterJobV2.class);
```

Comment:
write don't match with readFields

Reply:
I have conducted a compatibility test. The `read` of AlterJobV2 match the `write` of RollupJobV2 and SchemaChangeV2